src/configparser.cpp

```diff
@@ -96,15 +98,24 @@ bool ConfigParser::parse_file(const std::string& tmp_filename)
 	}
 
 	unsigned int linecounter = 0;
+	std::string multi_line_buffer{};
 	for (const auto& line : lines.value()) {
-		++linecounter;
-
-		LOG(Level::DEBUG, "ConfigParser::parse_file: tokenizing %s", line);
-
-		const std::string location = strprintf::fmt(_("%s line %u"), filename,
-				linecounter);
-
-		parse_line(line, location);
+		linecounter++;
+		const auto ln_del = line.rfind('\\');
+		if (ln_del != std::string::npos && ln_del == line.size()-1) {
```

Comment:
Why use `rfind` when you can check the last character directly with `!line.empty() && line.back() == '\\'`?